types/materialize-css/dropdown.d.ts

```diff
@@ -0,0 +1,145 @@
+/// <reference path="./common.d.ts" />
+
+declare namespace M {
+    class Dropdown extends Component<DropdownOptions> {
+        /**
+         * Get Instance
+         */
+        static getInstance(elem: Element): Dropdown;
+
+        /**
+         * Init Dropdown
+         */
+        static init(els: Element, options?: Partial<DropdownOptions>): Dropdown;
+
+        /**
+         * Init Dropdowns
+         */
+        static init(els: MElements, options?: Partial<DropdownOptions>): Dropdown[];
+
+        /**
+         * ID of the dropdown element
+         */
+        id: string;
+
+        /**
+         * The DOM element of the dropdown
+         */
+        dropdownEl: Element;
+
+        /**
+         * If the dropdown is open
+         */
+        isOpen: boolean;
+
+        /**
+         * If the dropdown content is scrollable
+         */
+        isScrollable: boolean;
+
+        /**
+         * The index of the item focused
+         */
+        focusedIndex: number;
+
+        /**
+         * Open dropdown
+         */
+        open(): void;
+
+        /**
+         * Close dropdown
+         */
+        close(): void;
+
+        /**
+         * While dropdown is open, you can recalculate its dimensions if its contents have changed
+         */
+        recalculateDimensions(): void;
+    }
+
+    interface DropdownOptions {
+        /**
+         * Defines the edge the menu is aligned to
+         * @default 'left'
+         */
+        alignment: string;
```

Comment:
From source code, it seems that this can only be 'left' or 'right', so define it as `'left' | 'right'` ?